RoddyCore/src/de/dkfz/roddy/core/ExecutionContextReaderAndWriter.groovy

```diff
@@ -342,7 +342,7 @@ class ExecutionContextReaderAndWriter {
                 if (split.length < 2) continue
 
                 String id = split[0]
-                JobState status = Roddy.getJobManager().parseJobState(split[1])
+                JobState status = null//Roddy.getJobManager().parseJobState(split[1]) TODO
```

Comment:
The Todo format is:

//TODO: ...


also please leave a whitespace after null.